Prox/Prox/AppDelegate.swift

```diff
@@ -13,10 +14,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     private var authorizedUser: FIRUser?
 
+    private lazy var remoteConfigCacheExpiration: TimeInterval = {
+        if AppConstants.isDebug {
+            // Refresh the config if it hasn't been refreshed in 60 seconds.
+            return 60
```

Comment:
The production builds will only refresh their configs every few hours or so.

The debug builds will likely want their configs refreshed on a edit-compile-run cycle of around 60 seconds.